symbolic-minidump/src/cfi.rs

```diff
@@ -1014,28 +1030,50 @@ pub struct CfiCache<'a> {
 impl CfiCache<'static> {
     /// Construct a CFI cache from an `Object`.
     pub fn from_object(object: &Object<'_>) -> Result<Self, CfiError> {
-        let buffer = AsciiCfiWriter::transform(object)?;
+        let mut buffer = vec![];
+        // TODO: for now when converting an object, we do *not* write a preamble, and output an
+        // `Unversioned` format. That way we never *write* a new versioned format yet, but support
+        // reading it
+        //write_preamble(&mut buffer, CFICACHE_LATEST_VERSION)?;
+        AsciiCfiWriter::new(&mut buffer).process(object)?;
+
         let byteview = ByteView::from_vec(buffer);
-        let inner = CfiCacheInner::V1(CfiCacheV1 { byteview });
+        //let inner = CfiCacheInner::Versioned(CFICACHE_LATEST_VERSION, CfiCacheV1 { byteview });
+        let inner = CfiCacheInner::Unversioned(CfiCacheV1 { byteview });
         Ok(CfiCache { inner })
     }
 }
 
+fn write_preamble<W: Write>(mut writer: W, version: u32) -> Result<(), io::Error> {
```

Comment:
Do you know why we're not receiving a warning about dead code here?

Reply:
it's used on line 1096

Reply:
Thanks :)

Reply:
It is still used when writing versioned caches, just that we never *create* such versioned caches.